docs/src/pages/index.tsx

```diff
@@ -1,41 +1,164 @@
-      description="Description will go into a meta tag in <head />">
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
+      // TODO: change title and description?
+      title={`Kargo - GitOps Promotion Tool`}
+      description="Stage to stage GitOps Promotion Made Easy With Kargo">
+      <main className='landing-page'>
+        <header className={styles.header}>
+          <h1 style={{margin: 0}}>Kargo</h1>
+          <span>Learn how to use Kargo for GitOps promotions of stages</span>
+          <div style={{marginTop: '24px'}}>
+            <Link to='/new-docs/user-guide/core-concepts'>
+              <button className='btn'>Overview</button>
+            </Link>
+            <Link to='/new-docs/user-guide/examples'>
+              <button className='btn btn-primary' style={{marginLeft: '24px'}}>Learn By Examples</button>
+            </Link>
+          </div>
+        </header>
+        <div className='container'>
+          <Section 
+            title={<><FaRocket /> Get Started</>}
+            nodes={[
+              {
+                id: 'install',
+                title: 'Installation',
+                description: (
+                  <>
+                    <Link to='/new-docs/operator-guide/basic-installation' className='highlight'>Basic Installation</Link>
+                    <br />
+                    <Link to='/new-docs/operator-guide/advanced-installation/advanced-with-helm' className='highlight'>With Helm</Link>
+                    <br />
+                    <Link to='/new-docs/operator-guide/advanced-installation/advanced-with-argocd' className='highlight'>With ArgoCD</Link>
+                  </>
+                ),
+                headerStyle: {
+                  background: '#1CAC77',
+                  color: 'white'
+                },
+                link: {
+                  to: '/new-docs/operator-guide/'
+                }
+              },
+              {
+                id: 'users-guide',
+                title: 'User\'s Guide',
+                description: 'Learn key concepts',
+                headerStyle: {
+                  background: '#FE7537',
+                  color: 'white'
+                },
+                link: {
+                  to: '/new-docs/user-guide/'
+                }
+              },
+              {
+                id: 'quickstart',
+                title: 'Quickstart',
+                description: 'Get your hands dirty with kargo by following quickstart',
+                headerStyle: {
+                  background: '#1DCECA',
+                  color: 'white'
+                },
+                link: {
+                  to: '/new-docs/quickstart/'
+                }
+              },
+            ]}
+          />
+
+          <Section 
+            title={<><FaTruck /> Advance in Kargo</>}
+            nodes={[
+              {
+                id: 'crd',
+                title: 'CRD Documentation',
```

Comment:
Can we change from CRD documentation and make it more generic, like "Reference Docs", and then have links for
- CRDs
- Expression Language
- Promotion Steps